test/images/hostexec/VERSION

```diff
@@ -1 +1 @@
-1.0
+1.1
```

Comment:
don't you have to change all the tests relying on this?

Reply:
Since we are changing `CMD`'s behavior, we should bump the version. If we don't, some clients will end up with different `CMD` behavior, depending on the local docker image cache state.

This makes it explicit at the cost of bumping the necessary dependent consumers (tests et al).

Reply:
more formally: If old and new behavior of `CMD` would equivalent, then no, a bump wouldn't be necessary, but this here looks like a totally different semantics.

Reply:
this LGTM. please build/push this image and update its references in e2e tests.

Reply:
New image has been pushed and tests have been updated. I think we are good to go.

Reply:
@ixdy, can you please take a look and apply the lgtm lable?